src/ovp_pipeline/materializers/crystal.py

```
@@ -0,0 +1,295 @@
+    import sqlite3
+
+    from ..knowledge_index import ensure_knowledge_db_current
+    from ..runtime import VaultLayout
+
+    ids = [oid for oid in object_ids if oid]
+    if not ids:
+        return []
+    try:
+        ensure_knowledge_db_current(vault_dir)
+    except Exception:
+        pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)